examples/server/README.md

```diff
@@ -274,6 +274,33 @@ And run it:
 node index.js
 ```
 
+Alternative script to test streaming mode (chunk splitting and error handling should be enhanced for production use):
+
+```javascript
+(async () => {
+    const response = await fetch("http://localhost:8080/completion", {
+        method: "POST",
+        body: JSON.stringify({
+            prompt: "To write an essay quickly",
+            n_predict: 256,
+            stream: true
+        })
+    })
+    for await (const chunk of response.body.pipeThrough(new TextDecoderStream("utf-8"))) {
+        for (const event of chunk.split(/(?<=\n\n)/v)) {
```

Comment:
This solution is OK for being a POC, but in reality, you may want to use `readline` because reverse proxy can cut the data chunk mod-way